Mothercell/Hive.IO/Hive.IO/DataHandling/Results.cs

```diff
@@ -29,6 +29,7 @@ public class Results
 
         #region General
         public double InterestRate { get; private set; }
+        public double TotalFloorArea { get; set; }
```

Comment:
shouldn't we set it to `private set;`?

Reply:
I was assuming it's a dummy anyway... I guess I'll just update that.